src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double side;
+    private double upperBase;
+    private double lowerBase;
+
+    public IsoscelesTrapezoid(double upperBase,double lowerBase, Color color) {
+        super(color);
+        this.upperBase = upperBase;
+        this.lowerBase = lowerBase;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((upperBase + lowerBase) / 2 * calculateHeight());
```

Comment:
Avoid rounding the result of the area calculation to maintain precision. Return the precise value instead.